vcpkg/readme/vcpkg-README.md

```diff
@@ -0,0 +1,347 @@
+need to install:
+
+- Debian, Ubuntu, popOS, and other Debian-based distributions:
+
+```sh
+$ sudo apt-get update
+$ sudo apt-get install build-essential tar curl zip unzip
+```
+
+- CentOS
+
+```sh
+$ sudo yum install centos-release-scl
+$ sudo yum install devtoolset-7
+$ scl enable devtoolset-7 bash
+```
+
+- Arch Linux based distributions:
+
+```sh
+$ sudo pacman -Syu base-devel curl zip unzip tar cmake ninja
+```
+
+For any other distributions, make sure you're installing g++ 6 or above.
+If you want to add instructions for your specific distro,
+[please open a PR][contributing:submit-pr]!
+
+## Installing macOS Developer Tools
+
+On macOS, the only thing you should need to do is run the following in your terminal:
+
+```sh
+$ xcode-select --install
+```
+
+Then follow along with the prompts in the windows that comes up.
+
+You'll then be able to bootstrap vcpkg along with the [quick start guide](#quick-start-unix)
+
+## Using vcpkg with CMake
+
+### Visual Studio Code with CMake Tools
+
+Adding the following to your workspace `settings.json` will make
+CMake Tools automatically use vcpkg for libraries:
+
+```json
+{
+  "cmake.configureSettings": {
+    "CMAKE_TOOLCHAIN_FILE": "[vcpkg root]/scripts/buildsystems/vcpkg.cmake"
+  }
+}
+```
+
+### Vcpkg with Visual Studio CMake Projects
+
+Open the CMake Settings Editor, and under `CMake toolchain file`,
+add the path to the vcpkg toolchain file:
+
+```
+[vcpkg root]/scripts/buildsystems/vcpkg.cmake
+```
+
+### Vcpkg with CLion
+
+Open the Toolchains settings
+(File > Settings on Windows and Linux, CLion > Preferences on macOS),
+and go to the CMake settings (Build, Execution, Deployment > CMake).
+Finally, in `CMake options`, add the following line:
+
+```
+-DCMAKE_TOOLCHAIN_FILE=[vcpkg root]/scripts/buildsystems/vcpkg.cmake
+```
+
+You must add this line to each profile.
+
+### Vcpkg as a Submodule with CMake
+
+When using vcpkg as a submodule of your project,
+you can add the following to your CMakeLists.txt before the first `project()` call,
+instead of passing `CMAKE_TOOLCHAIN_FILE` to the cmake invocation.
+
+```cmake
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake"
+  CACHE STRING "Vcpkg toolchain file")
+```
+
+This will still allow people to not use vcpkg,
+by passing the `CMAKE_TOOLCHAIN_FILE` directly,
+but it will make the configure-build step slightly easier.
+
+[getting-started:using-a-package]: https://learn.microsoft.com/vcpkg/examples/installing-and-using-packages
+[getting-started:git]: https://git-scm.com/downloads
+[getting-started:cmake-tools]: https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools
+[getting-started:linux-gcc]: #installing-linux-developer-tools
+[getting-started:macos-dev-tools]: #installing-macos-developer-tools
+[getting-started:macos-brew]: #installing-gcc-on-macos
+[getting-started:macos-gcc]: #installing-gcc-on-macos
+[getting-started:visual-studio]: https://visualstudio.microsoft.com/
```

Comment:
Let's make sure that all of this is covered in the new **Get Started** articles and remove these sections from the README.

Reply:
I've removed a bunch of sections that were already included in the get-started articles. I've left information I felt wasn't in those articles yet. I'm working on trying to incorporate some of the developer tools required for UNIX platforms